test/transactions/index.js

```diff
@@ -17,45 +17,45 @@ import transaction from '../../src/transactions/index';
 describe('transactions', () => {
 	describe('exports', () => {
 		it('should have the create transfer transaction function', () => {
-			return transaction.should.have.property('transfer').of.type('function');
+			return transaction.should.have.property('transfer').of.a('function');
```

Comment:
`which.is.a`?

Reply:
More like neutral language! 👍

Reply:
To be consistent with other places, changed to `and.be.a('function');`

Reply:
OK, I'd like to move more in the direction of natural-sounding assertions generally speaking though.